e2e-tests/adapters/cypress/e2e/headers.cy.ts

```diff
@@ -1,5 +1,7 @@
+import { applyTrailingSlashOption } from "../../utils"
```

Comment:
Those changes in this test are not related to sharp changes.

I'm not exactly what changed, but without the changes we are now making header assertions on 301 redirects instead of 200 html responses. Those changes ensure we do direct visit to expected html route thus avoiding 301 redirect (that's redirect for trailing slash) that is messing our assertions